front-end/src/components/ResetPassword/ModalWindow.jsx

```diff
@@ -14,11 +14,13 @@ import { CurrentUserContext } from 'context/AppProvider';
 import { changePassword } from 'services/userService';
 import { REGEX_PASSWORD } from 'helpers/regex';
 import useInput from 'hooks/useInput';
+import Loader from 'components/common/Loader/Loader';
+import { withSnackbar } from 'components/withSnackbar/withSnackbar';
 
 const style = {
   position: 'absolute',
   top: '50%',
-  right: '416px',
+  left: '60%',
```

Comment:
Are you sure that 60% is aligning by center on different screen sizes?

Reply:
changed to be 50%, so at all screens modal will be on centre of the screen